modules/cnv-networking-wizard-fields-web.adoc

```diff
@@ -13,10 +13,10 @@
 |Name | Description
 
 |Name
-|Name for the NIC.
+|Name for the Network interface.
 
-|MAC Address
-|MAC address for the network interface. If a MAC address is not specified, an ephemeral address is generated for the session.
+|Model
+|Driver for the Network interface or model for the Network interface.
```

Comment:
Same as above -- lowercase or `Network Interface Card`

Reply:
Made it consistent - Network Interface Card.